server/index.js

```diff
@@ -2,11 +2,36 @@ require('dotenv/config');
 const express = require('express');
 const errorMiddleware = require('./error-middleware');
 const staticMiddleware = require('./static-middleware');
+const pg = require('pg');
+
+const db = new pg.Pool({
+  connectionString: process.env.DATABASE_URL,
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
 
 const app = express();
 
 app.use(staticMiddleware);
 
+app.get('/api/images', (req, res, next) => {
+
+  const sql = `
+     select  "photoId",
+             "name",
+             "url",
+             "homepage"
+       from  "photos"
+       join  "projects" using ("projectId")
+`;
```

Comment:
If you include a where clause here, then you can probably eliminate your conditional statements on the front end.